libs/datasets/dataset_utils.py

```diff
@@ -284,6 +285,100 @@ def summarize(data, aggregate_level, groupby):
     print(index_size[index_size > 1])
 
 
+def _clear_common_values(existing_df, data_source, index_fields, column_to_fill):
+    """For labels shared between existing_df and data_source, clear column_to_fill of existing_df inplace."""
```

Comment:
I'm not sure labels is immediately obvious to me? Is a label an index row?

Reply:
Yes. Adding more docs.